src/terminal/adapter/DispatchTypes.hpp

```diff
@@ -280,7 +280,7 @@ namespace Microsoft::Console::VirtualTerminal
         }
 
         template<typename T>
-        bool for_each(const T&& predicate) const
+        void for_each(const T&& predicate) const

```

Comment:
Before you say anything: I slightly regret going down this path. I pulled it through because of the sunken cost fallacy.